src/components/application_manager/test/help_prompt_manager_test.cc

```diff
@@ -180,8 +180,9 @@ void HelpPromptManagerTest::SetUp() {
       app_mngr_,
       HmiState::STATE_ID_REGULAR);
 
-  std::string path = file_system::CreateDirectory("storage");
-  file_system::CreateFile(path + "/" + "certificate");
+  std::string path("storage");
```

Comment:
@SKobziev Can be const?

Reply:
@AByzhynar see 4734f0a5a55c3e2bc2ec500a55244d03ce3fa358